src/js/controller/controls-loader.js

```diff
@@ -1,7 +1,7 @@
 define([
 
 ], function() {
-    let controls = null;
```

Comment:
The loader is not part of jwplayer.controls.js so I moved it out of the controls dir. No babel loading there until after #1927 is merged.